### PR TITLE
Search Test coverage to 100%

### PR DESCRIPTION
**Pull request by sheela-svmx**
screen shot 2018-04-16 at 10 14 37 pm

### CODE REVIEW

**Comment by hanzge** [comment]
On your screenshot, the  branch coverage is still 88.46%. Please check if there're any `if` statements, `a ? b : c` operations, etc. not tested.

**Comment by sheela-svmx** [comment]
ooh will take a look, I never understood what the Brach 88% was// rest all was 100%, so is it the if statements?

**Comment by sheela-svmx** [comment]
updated @hanzge

File: lib/components/Search/Search.test.js [hunk 1]
```diff
@@ -137,4 +145,57 @@ describe('Search', () => {
...
+      },
+    };
+    const input = wrapper.find('Input');
+    input.simulate('change', mockEvent);
+    input.simulate('keyDown', { keyCode: 13 });
+    expect(onKeyDownFn.mock.calls.length).toEqual(0);
+  });
+
+  it('should simulate the isFocusedInComponent function', () => {
+    const utilModule = require('./../utils');
```
Comment (mallik-svmx) [inline lib/components/Search/Search.test.js]: Can't we just use ES6 import Vs. commonJS require?

**Pull request merged**